Assets/OneClassOneGame.cs

```diff
@@ -15,15 +15,17 @@ public class OneClassOneGame : MonoBehaviour
     public bool IsStartedBool;
     public bool DonateClickBool;
 
+    System.Random rand = System.Random();
     public void Update()
     {
-        if (IsStartedBool == false)
+        if ((IsStartedBool != null) ? not(IsStartedBool): false)
         {
             CurrentPointsFloat = StartPointsFloat;
             IsStartedBool = true;
         }
 
-        if (DonateClickBool == true)
+        (rand.Next()%100==0) ? DonateClickBool = true : DonateClickBool = not(!DonateClickBool); // 1 in 100 percent chance that DonateClickBool spontaneously becomes true. mmm yes money.
```

Comment:
Hello, thanks for valuing business ideas, but it probably would be rejected
Please increase chance a bit (up to 146% for example), so this functionality would be useful